### DISCUSSION TITLE
How to hide validation message container

### DISCUSSION THREAD

**Discussion by sShakar** [quasarframework/quasar#14401] [General - Components / Directives / etc]
Hello,
I would like to have validation rules for my input fields with the red indicators and the error-icon but I do not want to have an error message container. Would you please let me know how to remove the message container which have the classes `q-field__bottom row items-start q-field__bottom--animated`.
I have tried to override/extend the existing classes by adding `display: none;` but it's not working.

**Comment by pdanpdan** [comment]
`hide-bottom-space`

**Reply by sShakar** [reply]
thank you so much for the answer, that helps in a lot of situations.
but there is still the problem of the empty container showing below the field when the validation fails.

![image](https://user-images.githubusercontent.com/70109216/190422095-354da3f9-27c0-45e8-b7d7-8368b20f1df6.png)

**Comment by pdanpdan** [comment]
Please make a codepen

**Reply by sShakar** [reply]
It seems like the validation rules doesn't work in codepen

**Reply by rstoenescu** [reply]
They definitely do work in codepen.
